bitnami/pytorch/templates/statefulset.yml

```diff
@@ -0,0 +1,163 @@
+{{- if .Values.pytorch.distributed.enabled }}
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: {{ include "pytorch.fullname" . }}-worker
+  labels:
+    app.kubernetes.io/name: {{ include "pytorch.name" . }}
+    helm.sh/chart: {{ include "pytorch.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "worker"
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "pytorch.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: "worker"
+  replicas: {{ sub .Values.pytorch.distributed.worldSize 1 }}
+  template:
```

Comment:
No need of `serviceName` and "headless" services?

Reply:
I don't think so.